test/unit/test_auth_oauth.py

```diff
@@ -5,14 +5,16 @@
 
 from __future__ import annotations
 
-from snowflake.connector.auth_oauth import AuthByOAuth
+try:  # pragma: no cover
+    from snowflake.connector.auth import AuthByOAuth
+except ImportError:
+    from snowflake.connector.auth_oauth import AuthByOAuth
```

Comment:
why we still try to import from the old module first?

Reply:
`snowflake.connector.auth` is the new module

Reply:
oh, is it because we have the old driver test, so we still need to import the old ones?

Reply:
Yup, while old driver tests don't run this file. If the name resolution fails that'll still crash the whole `pytest` run